src/main/java/org/eclipse/cargotracker/infrastructure/messaging/JMSResourcesSetup.java

```diff
@@ -0,0 +1,56 @@
+package org.eclipse.cargotracker.infrastructure.messaging;
+
```

Comment:
Let us kindly move this back to XML. These settings, particularly the resource adapter, varies from one provider to the other, so we will need to use Maven substitution in the web.xml.